from_variants_impl/Cargo.toml

```diff
@@ -13,9 +12,9 @@ proc-macro = true
 
 [dependencies]
```

Comment:
Should we bump the rest of these to their latest versions?

Reply:
They are already at their latest versions. `proc-macro2/syn/quote` are developed by David Tolnay. And he makes his crates extra-stable. He knows how much burden it is for the ecosystem to keep up-to-date. That's why he always does only patch releases.
You can see for yourself:
- [`proc-macro2` crate](https://docs.rs/proc-macro2/latest/proc_macro2/): `1.0.36`
- [`quote` crate](https://docs.rs/quote/latest/quote/): `1.0.15`
- [`syn` crate](https://docs.rs/syn/latest/syn/): `1.0.86`